### PR TITLE
fixed node incompatibility warning

### PR DESCRIPTION
**Pull request by dereknelson**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -44,4 +44,4 @@
     "yargs": "^14.0.0",
     "yargs-unparser": "^1.5.0"
   }
-}
+}
```
Comment (bradbyte) [inline package.json]: Can you include the end of file line here? There's a `.editorconfig` file in the src that contains the rules. If you could also fixup these commits to be a single change too that would be great. Thank you.

**Review by bradbyte — CHANGES_REQUESTED** [review]

**Comment by dereknelson** [comment]
@bradbumbalough done :)

**Comment by bradbyte** [comment]
Thank you!

**Comment by bradbyte** [comment]
Released in `0.0.2` 🎉 .

**Pull request merged**